core/css.js

```diff
@@ -28,16 +28,14 @@ let injected = false;
 /**
  * Add some CSS to the blob that will be injected later.  Allows optional
  * components such as fields and the toolbox to store separate CSS.
- * The provided array of CSS will be destroyed by this function.
- * @param {!Array<string>} cssArray Array of CSS strings.
+ * @param {string} cssContent Multiline CSS string.
  */
-const register = function(cssArray) {
+const register = function(cssContent) {
```

Comment:
Can you actually update this function to take in either a `string` or an `Array`? If an Array is passed in we can just convert it to a string and then use the [warn](https://github.com/google/blockly/blob/7b3a3aa74379465690dfed9c6c9f766809cf95c6/core/utils/deprecation.js#L31) method to tell the developer the API will change soon. The deletion date should be September 2022.

Reply:
I took care of this.